src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1,4 +1,95 @@
 featurizers:
+  # ----------------------------------------------------------------------
+  - name: AnalyticalRollingWindowFeaturizer
+    estimator_name: AnalyticalRollingWindowEstimator
+    release_version: 0.4.0
+    num_output_columns: 1
+    has_dynamic_output: false
+    description: |-
+        Calculates data based on a rolling window. Currently supports mean. Works for general time series data sets.
+        
+        Input type for this featurizer is a tuple of time point of current row, the grain columns, and target value column.
+
+        C++-style pseudo signature:
+        template <typename T> std::vector<double> execute(std::tuple<std::vector<std::string> const &, T const &> value);
+
+        Examples:
+            A simple example would be horizon = 1, window size = 2, and we want to take the mean.
+
+            +----------------+-----------+-------+-------------------+
+            |time(YYYY-MM-DD)| grain     | target| target_mean       |
+            +================+===========+=======+===================+
+            | 2017-01-01     | A         | 10    | [NAN]             |
+            +----------------+-----------+-------+-------------------+
+            | 2017-02-01     | A         | 4     | [10]              |
+            +----------------+-----------+-------+-------------------+
+            | 2017-03-01     | A         | 6     | [7]               |
+            +----------------+-----------+-------+-------------------+
+            | 2017-04-01     | A         | 11    | [5]               |
+            +----------------+-----------+-------+-------------------+

```

Comment:
Updates these tables now that we are no longer taking a datetime as input.

Reply:
Removed the date column.